app.py

```diff
@@ -94,6 +95,11 @@ def ec2_action_():
     def get_cluster_config_():
         return get_cluster_config()
 
+    @app.route("/manager/get_cluster_template")
```

Comment:
suggestion: cluster templates are plain strings and can probably be inlined in the JS bundle as they add few extra bytes (nothing comparable to third party modules). What I suggest is to remove this extra configuration management and use something like [next-plugin-yaml](https://www.npmjs.com/package/next-plugin-yaml) to inline the templates frontend side. This has several benefits:
- no new route backend side
- no new custom requests frontend side and error management
- no manual transform of yaml -> js: this gets handled automatically by `next-plugin-yaml`
- simplified testing: no need to manually edit URL to fetch template, simply edit the files and see the result in the browser

Reply:
I like this approach! I think the only downside is we can't update between releases then right?

Reply:
Yes, correct, but we can start small by making those files static and then, if we have to for some reasons, write some code to add dynamic imports from cloud. Anyway, I think that if we invest in testing, build times and automated deployments it is going to be much easier to release something thus static files should be ok imho.